cmd/health/health_status.go

```diff
@@ -4,15 +4,19 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/stackvista/stackstate-cli/internal/util"
-
 	"github.com/spf13/cobra"
 	"github.com/stackvista/stackstate-cli/generated/stackstate_api"
 	"github.com/stackvista/stackstate-cli/internal/common"
 	"github.com/stackvista/stackstate-cli/internal/di"
 	"github.com/stackvista/stackstate-cli/internal/printer"
 )
 
+const (
+	// Magic numbers are frowned upon, so I'm introducing these two smelly ding-dongs because I don't give a fuck anymore.
+	TWO   = 2
```

Comment:
There is a `//nolint:gomnd` annotation you can add to the line, method or file ;)

Reply:
FIxed